src/transformState.js

```diff
@@ -5,7 +5,22 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach(action => {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
+      default :
+        Object.keys(state).forEach(key => {
+          delete state[key];
+        });
+    }
```

Comment:
believe that it should be default behaviour
what if someone will pass invalid data - state will be cleared by default
[usefulLink](https://www.quora.com/What-is-the-purpose-of-default-in-a-switch-case)
[usefulLink](https://softwareengineering.stackexchange.com/questions/181459/is-it-necessary-to-add-the-default-case-while-using-switch-cases)